SDWebImage/SDWebImageDownloaderOperation.m

```diff
@@ -429,7 +380,7 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didComp
             if (self.imageData) {
                 UIImage *image = [UIImage sd_imageWithData:self.imageData];
                 NSString *key = [[SDWebImageManager sharedManager] cacheKeyForURL:self.request.URL];
-                image = [self scaledImageForKey:key image:image];
```

Comment:
Maybe this is no need to change and this allow potential override from the user(Though this is not recommaned).